Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -52,10 +56,16 @@ block FirstOrderAMIGO
     "Initial value of derivative output"
     annotation (Dialog(tab="Advanced",group="Initialization",enable=controllerType == Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Types.SimpleController.PID));
   parameter Boolean reverseActing=true
-    "Set to true for reverse acting, or false for direct acting control action";
+    "Set to true for reverse acting, or false 
+     for direct acting control action";
   parameter Real y_reset=xi_start
-    "Value to which the controller output is reset if the boolean trigger has a rising edge"
+    "Value to which the controller output is reset 
+     if the boolean trigger has a rising edge"
     annotation (Dialog(group="Integrator reset"));
+  parameter Real SetHys = 0.001*r
```

Comment:
`setHys`, not `SetHys`. Also, it might be too small, 0.001*r? we are thinking like 5% of the typical error range?

Reply:
Fixed the naming issue and used 5%.